Sail/src/Sail/entities/systems/Graphics/CrosshairSystem.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "../BaseComponentSystem.h"
+#include "Sail/Application.h"
+
+class Entity;
+class CrosshairComponent;
+
+/*
+Crosshair is rendered in InGameGui.cpp
+*/
+
+class CrosshairSystem : public BaseComponentSystem {
+public:
+	CrosshairSystem();
+	~CrosshairSystem();
+
+	void update(float dt);
+
+private:
+	SettingStorage* m_settings = nullptr;
+
+	void applySettings(CrosshairComponent* c);
+	void alterCrosshair(Entity* e, float dt);
+};
```

Comment:
Application.h does not need to be included in CrosshairSystem.h, since the only thing used from it is a SettingStorage pointer which can be forward declared?

Reply:
Fixed